apps/docs/generator/index.ts

```diff
@@ -3,28 +3,42 @@ import CliGenerator from './cli'
 import ConfigGenerator from './config'
 import SdkGenerator from './sdk'
 import LegacyGenerator from './legacy'
-import * as minimist from 'minimist'
+import { parseArgs } from 'node:util'
+const assert = require('node:assert')
```

Comment:
Any reason this needed to be `require` over `import`?

Reply:
There were some weird type errors + execution issues, but I now found a working way with es6 imports